web/src/elements/table/Table.ts

```diff
@@ -107,8 +100,7 @@ export abstract class Table<T> extends AKElement {
         return false;
     }
 
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    renderExpanded(item: T): TemplateResult {
+    renderExpanded(_item: T): TemplateResult {
```

Comment:
I think I left the variable name as `item` here as that's the default when inheriting from the class

Reply:
Good point; I've put it back the way it was.

I was trying to use the "unused variable in abstract code" idiom here, and it shouldn't matter what the name is in the child code, only the position.  But leaving the name as-is and commenting out the check is probably better for readability in the long-run.